test/app/android/app/src/main/AndroidManifest.xml

```diff
@@ -39,6 +39,7 @@
     <service
       android:enabled="true"
       android:name="com.twiliovoicereactnative.IncomingCallNotificationService"
+      android:foregroundServiceType="microphone"
```

Comment:
Let's make sure we port this to the Reference App and recommend it to projects that use our SDK.

Reply:
@mhuynh5757 Sure, create a ticket for that and I will, btw where is the code for that project?

Reply:
This is the repo for the Reference App https://github.com/twilio/twilio-voice-react-native-app